purchase_manual_delivery/report/report_stock_forecast.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="purchase_report_product_product_replenishment"
+        inherit_id="stock.report_product_product_replenishment"
+    />
+</odoo>
```

Comment:
Is this override strictly necessary?

Reply:
As far as I can tell: No. I tested it and it seems to work fine without it. I kept it in the code to see that this template or rather a part of it is gets changed. Odoo has the same override in its modules. See https://github.com/odoo/odoo/blob/16.0/addons/purchase_stock/report/report_stock_forecasted.xml
I can remove it if it is a problem.

Reply:
Looks like the file in Odoo is a remainder from this refactoring: https://github.com/odoo/odoo/commit/2bece51b24a960527c0609c4192e177f710e9bfd#diff-be21cb1e02bbe01d1a5bc4701e265ec107ff9717d3f272071c1bfb4822351196

I removed it in the 17.0 migration without any issue. You can do the same here.

Reply:
Gotcha. I removed it. 
Thank you very much.